Makefile

```diff
@@ -174,19 +174,23 @@ install-konduit: ## Install the konduit script, for accessing backend services
 
 aks-console: get-cluster-credentials
 	$(if $(PULL_REQUEST_NUMBER), $(eval export APP_ID=review-$(PULL_REQUEST_NUMBER)) , $(eval export APP_ID=$(CONFIG_LONG)))
-	kubectl -n ${NAMESPACE} exec -ti --tty deployment/npq-registration-${APP_ID}-web -- /bin/sh -c "cd /app && bundle exec rails c"
+	kubectl -n ${NAMESPACE} exec -ti --tty deployment/npq-registration-${APP_ID}-worker -- /bin/sh -c "cd /app && bundle exec rails c --sandbox"
+
+aks-rw-console: get-cluster-credentials
```

Comment:
worth updating `docs/connecting-to-azure.md` to mention these new commands

Reply:
Done :+1: